setup.cfg

```diff
@@ -17,15 +17,15 @@ search = APP_VERSION ?= {current_version}
 replace = APP_VERSION ?= {new_version}
 
 [bumpversion:file:HISTORY.rst]
-search = 
-	Unreleased
-	---------------------
-replace = 
-	Unreleased
-	---------------------
-	
-	{new_version} ({now:%%Y-%%m-%%d})
-	---------------------
+search =
+	`Unreleased <https://github.com/Ouranosinc/CanarieAPI/tree/master>`_ (latest)
+	------------------------------------------------------------------------------------
+replace =
+	`Unreleased <https://github.com/Ouranosinc/CanarieAPI/tree/master>`_ (latest)
+	------------------------------------------------------------------------------------
+
+	`{new_version} <https://github.com/crim-ca/weaver/tree/{new_version}>`_ ({now:%%Y-%%m-%%d})
```

Comment:
Why refer to Weaver tree in this CanarieAPI setup.cfg?

Reply:
Oups, nice catch. copy-paste error. Thanks 👍